### PR TITLE
Add `npm run develop`

### PR DESCRIPTION
**Pull request by tomekwi**
`npm run develop`, change files, and see wonders.

### CODE REVIEW

**Comment by stoeffel** [comment]
Nice :+1:

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
+    "develop": "nodangel --watch src --watch test --exec 'npm test'",
```
Comment (stoeffel) [inline package.json]: `--watch tests`

**Comment by tomekwi** [comment]
Yeah, I’ve just been wondering why it wouldn’t work for tests :)

**Comment by stoeffel** [comment]
:rocket:

**Pull request merged**